src/transactionNotifier/TransactionNotifier.scss

```diff
@@ -64,7 +64,9 @@
 .description {
   color: $grey-light;
 }
-
+.link {
+  color: #80D8FF;
```

Comment:
Can you extract the color in the variable.scss? I use colorhexa.com for proper naming :)

Reply:
I always search for colors in variables.scss first, but for some reason it did not work this time...